oak_runtime/src/node/http/mod.rs

```diff
@@ -56,6 +60,16 @@ use tokio_rustls::TlsAcceptor;
 #[cfg(test)]
 pub mod tests;
 
+#[allow(clippy::enum_variant_names)]
```

Comment:
It may actually make sense to remove the `Error` suffix from the variants IMO, but up to you.